source/runner/templates/github-issue-template.ts

```diff
@@ -33,12 +33,13 @@ function table(name: string, emoji: string, violations: Array<Violation>): strin
  * @param {DangerResults} results Data to work with
  * @returns {string} HTML
  */
+// TODO update function to support new DangerResults.markdowns type
 export function template(results: DangerResults): string {
   return `
 ${table("Fails", "no_entry_sign", results.fails)}
 ${table("Warnings", "warning", results.warnings)}
 ${table("Messages", "book", results.messages)}
-${results.markdowns.join("\n\n")}
+${table("Markdowns", "information_source", results.markdowns)}
```

Comment:
Actually, I'm going to change this back to just displaying markdown as it's provided, based on this comment - https://github.com/danger/danger-js/issues/109#issuecomment-274298803.